integration_test/tests/common/mod.rs

```diff
@@ -126,30 +121,50 @@ pub fn sql_return_to_string(return_code: SqlReturn) -> String {
 }
 
 #[allow(dead_code)]
-// Allocates new environment variable
-pub fn allocate_env() -> Result<HEnv> {
+/// Setup flow that connects and allocates a statement. This allocates a new
+/// environment handle, sets the ODBC_VERSION environment attribute, connects
+/// using the provided (or default) URI, and allocates a statement. The flow
+/// is:
+///     - SQLAllocHandle(SQL_HANDLE_ENV)
+///     - SQLSetEnvAttr(SQL_ATTR_ODBC_VERSION, SQL_OV_ODBC3)
+///     - SQLAllocHandle(SQL_HANDLE_DBC)
+///     - SQLDriverConnectW
+///     - SQLAllocHandle(SQL_HANDLE_STMT)
+pub fn default_setup_connect_and_alloc_stmt(
+    odbc_version_value: AttrOdbcVersion,
+) -> (HEnv, HDbc, HStmt) {
+    let env_handle = allocate_env(odbc_version_value);
+    let (conn_handle, stmt_handle) = connect_and_allocate_statement(env_handle, None);
+
+    (env_handle, conn_handle, stmt_handle)
+}
+
+#[allow(dead_code)]
+/// Allocates new environment variable and sets the ODBC_VERSION environment
+/// attribute to the provided odbc_version.
+pub fn allocate_env(odbc_version: AttrOdbcVersion) -> HEnv {
     let mut env: Handle = null_mut();
 
     unsafe {
-        match SQLAllocHandle(
-            HandleType::SQL_HANDLE_ENV,
-            null_mut(),
-            &mut env as *mut Handle,
-        ) {
-            SqlReturn::SUCCESS => (),
-            sql_return => return Err(Error::HandleAllocation(sql_return_to_string(sql_return))),
-        }
-        match SQLSetEnvAttr(
-            env as HEnv,
-            EnvironmentAttribute::SQL_ATTR_ODBC_VERSION,
-            AttrOdbcVersion::SQL_OV_ODBC3.into(),
-            0,
-        ) {
-            SqlReturn::SUCCESS => (),
-            sql_return => return Err(Error::SetEnvAttr(sql_return_to_string(sql_return))),
-        }
+        assert_eq!(
```

Comment:
I think that the former construction of errors was done on purpose.
The idea I believe was to know what failed without having to look at the line number. With these asserts, we will only see something like 
```
assertion `left == right` failed
  left: true
 right: false
```
Even if rely on assert now, we should add a more descriptive message.

Reply:
Updated all new asserts to include the same detailed error message as the existing ones 👍